src/main/java/seedu/edulog/ui/LessonListPanel.java

```diff
@@ -41,6 +42,7 @@ protected void updateItem(Lesson lesson, boolean empty) {
                 setGraphic(null);
                 setText(null);
             } else {
+                logger.fine("Updating LessonListViewCell with lesson: " + lesson);
```

Comment:
Works well with toString for lesson

Reply:
Good Logger use! I think an assertion in this method could work as well :)